llvm/test/CodeGen/AMDGPU/bitcast-vector-extract.ll

```diff
@@ -73,7 +73,7 @@ define amdgpu_kernel void @store_bitcast_constant_v8i32_to_v16i16(ptr addrspace(
 define amdgpu_kernel void @store_value_lowered_to_undef_bitcast_source(ptr addrspace(1) %out, i64 %a, i64 %b) #0 {
   %undef = call i64 @llvm.amdgcn.icmp.i64(i64 %a, i64 %b, i32 999) #1
   %bc = bitcast i64 %undef to <2 x i32>
-  store volatile <2 x i32> %bc, ptr addrspace(1) %out
+  store <2 x i32> %bc, ptr addrspace(1) %out
```

Comment:
Don't know why these tests are losing volatile

Reply:
In this case, the removal of the volatile preserves the original intent of the test, that there is no `store_dword`.  I'm not really sure what we should here since this test is explicitly checking for the absence of a store.